src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+import Todo from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={classNames('TodoInfo', {
+        'TodoInfo--completed': todo.completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">
+        {todo.title}
+      </h2>
+
+      <UserInfo user={todo.user} />
```

Comment:
find can return undefined
<img width="486" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/70220667/11f65ae6-7a87-41c4-a96c-6e06bf17baa1">


let's handle this case here
```suggestion
      {todo.user && <UserInfo user={todo.user} />}
```